### ISSUE TITLE
Bug: entryPoint is not resolved in pnpm monorepo

### ISSUE THREAD

**Issue by YusukeSano** [schoero/eslint-plugin-readable-tailwind#224]
## Bug summary

In a pnpm monorepo, if `tailwindcss` is installed only in a package-level `node_modules` (not hoisted to the root), `eslint-plugin-better-tailwindcss` fails to resolve the entryPoint.

```
⚠️ Warning: Tailwind CSS is not installed.
```

#### How to reproduce
- pnpm monorepo
- `tailwindcss` installed only in a package (not at root)
- Run ESLint from the monorepo root
- See: [YusukeSano/eslint-plugin-better-tailwindcss-monorepo](https://github.com/YusukeSano/eslint-plugin-better-tailwindcss-monorepo) for a minimal reproduction

#### Expected
- entryPoint should be resolved even if Tailwind is only installed at the package level

#### Actual
- entryPoint is not detected, and no error or warning is shown

**Comment by schoero** [comment]
Thank you for the reproduction, that really helped.

Fixing this required a pretty large refactoring. Tailwind will now be imported relative to the `entryPoint` and no longer from the current working directory.

The fix will be included in the next major version.

I just released a beta version for the next major release. 

Note that the beta version already contains some [breaking changes](https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/v4/CHANGELOG.md#v400-beta0) and more breaking changes will be added before the final release.

You can install the beta version using the following command:

```sh
pnpm add eslint-plugin-better-tailwindcss@beta
```

**Comment by yf-hk** [comment]
@schoero Can you please fix that in the next version? Manually adding valibot @valibot/to-json-schema doesn't seem to be a good solution
